site/src/app/roadmap/toolbar/Header.scss

```diff
@@ -84,7 +59,7 @@
         padding-inline: 8px 6px;
```

Comment:
These custom styles might be something to consider changing or removing since we're now using the full MUI button shape. The buttons for mobile in the screenshots look a bit off because they don't use the same padding ratios as other MUI buttons (i.e. Add Course)